server/models/memberAccount.js

```diff
@@ -55,67 +55,67 @@ module.exports = (DataTypes, sequelize) => {
             allowNull: false
         },
         homeToShareDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         isInterestedInBuyingHome: {
           type: DataTypes.BOOLEAN,
           allowNull: false
         },
         interestInBuyingHomeDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         isReligionImportant: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         religionDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         isDietImportant: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         dietDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         hasHealthMobilityIssues: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         healthMobilityIssuesDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         hasAllergies: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         allergiesDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         hasPets: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         petsDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         isSmoker: {
             type: DataTypes.BOOLEAN,
             allowNull: false
         },
         smokingDescription: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(100)
         },
         numRoommates: {                  // -1 means any number
             type: DataTypes.INTEGER,
             allowNull: false
         },
         workStatus: {
-            type: DataTypes.STRING,
+            type: DataTypes.STRING(100),
             allowNull: false
         },
         bio: {
-            type: DataTypes.STRING
+            type: DataTypes.STRING(2000)
```

Comment:
we should probably put this "max" string count in the bio label so people know how many words / characters they can enter

Reply:
This is going to be done in #819

Reply:
We need to add server and client side validation as well, I think the DB would probably crash everything lol (see #818 and #819 )